numba/tests/test_remove_dead.py

```diff
@@ -78,8 +100,91 @@ def seed_call_exists(func_ir):
             return False
 
         test_ir = compiler.run_frontend(call_np_random_seed)
-        remove_dead(test_ir.blocks, test_ir.arg_names)
+        remove_dead(test_ir.blocks, test_ir.arg_names, test_ir)
         self.assertTrue(seed_call_exists(test_ir))
 
+    def test_alias_ravel(self):
+        def func(A, i):
+            B = A.ravel()
+            B[i] = 3
+
+        A1 = np.arange(6).reshape(2,3)
+        A2 = A1.copy()
+        i = 0
+        pfunc = self.compile_parallel(func, (numba.typeof(A1), numba.typeof(i)))
+
+        func(A1, i)
+        pfunc(A2, i)
+        np.testing.assert_array_equal(A1, A2)
+
+    def test_alias_flat(self):
+        def func(A, i):
+            B = A.flat
+            B[i] = 3
+
+        A1 = np.arange(6).reshape(2,3)
+        A2 = A1.copy()
+        i = 0
+        pfunc = self.compile_parallel(func, (numba.typeof(A1), numba.typeof(i)))
+
+        func(A1, i)
+        pfunc(A2, i)
+        np.testing.assert_array_equal(A1, A2)
+
+    def test_alias_transpose1(self):
+        def func(A, i):
+            B = A.T
+            B[i,0] = 3
+
+        A1 = np.arange(6).reshape(2,3)
+        A2 = A1.copy()
+        i = 0
+        pfunc = self.compile_parallel(func, (numba.typeof(A1), numba.typeof(i)))
+
+        func(A1, i)
+        pfunc(A2, i)
+        np.testing.assert_array_equal(A1, A2)
+
+    def test_alias_ctypes(self):
```

Comment:
This is the test that needs the BLAS guard `@needs_blas`, I think the rest are ok?

Reply:
Done. Yes, only this one needs BLAS.